lean4-infoview/src/infoview/rpcSessions.tsx

```diff
@@ -96,13 +96,14 @@ export class RpcSessions implements Disposable {
 
     private async sessionAt(uri: DocumentUri): Promise<RpcSession | undefined> {
         if (this.#connected.has(uri)) return this.#connected.get(uri)
-        else if (this.#connecting.has(uri)) return this.#connecting.get(uri)
+        else if (this.#connecting.has(uri)) return await this.#connecting.get(uri)
```

Comment:
The `await` shouldn't be required here, or am I missing something?

Reply:
Hmm I'm not sure. If you do `return p` where `p : Promise<T>` in an async function  does the compiler automatically squash the `Promise<Promise<T>>` to a `Promise<T>`? I guess it must do.

Reply:
It's not required, I just put it in for clarity.